src/test/java/edu/ucsb/courses/controllers/BasicSearchControllerTests.java

```diff
@@ -28,12 +37,20 @@
 @Import(SecurityConfig.class)
 public class BasicSearchControllerTests {
 
+    private final Logger logger = LoggerFactory.getLogger(BasicSearchControllerTests.class);
+
     @Autowired
     private MockMvc mockMvc;
 
     @MockBean
     private UCSBCurriculumService ucsbCurriculumService;
 
+    // @MockBean
+    // private DownloadCsvService downloadCsvService;
```

Comment:
Is it necessary to keep this commented code in the final

Reply:
^^^ What he said